libs/utils/Cargo.toml

```diff
@@ -54,6 +54,8 @@ bytes.workspace = true
 criterion.workspace = true
 hex-literal.workspace = true
 tempfile.workspace = true
+serde_assert.workspace = true
+claims.workspace = true
```

Comment:
I don't think this was actually used anywhere?

Reply:
yes, I think it is redundant crate.
Will remove it.

Reply:
Done https://github.com/neondatabase/neon/pull/5335/commits/c6fae9c23c806c39e4cdd355441538506d399f7b